File: 1/one.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
)

func main() {
	readFile, _ := os.Open("input.txt");
	fileScanner := bufio.NewScanner(readFile)
	max, total := 0, 0
	for fileScanner.Scan() {
		line := fileScanner.Text();
		intLine, err := strconv.Atoi(line)
		if(err == nil){
			total += intLine
		}else{
			if total > max {
				max = total
			}
			total = 0
		}

	}
	readFile.Close();
	fmt.Println(max);
}

File: 1/one.js
const fs = require('fs');

const contents = fs.readFileSync('./input.txt', 'utf8');
var total = 0;
var max = 0;
contents.split(/\r?\n/).forEach(line => {
	if(line == ''){
		if (total > max){
			max = total
		}
		total = 0;
	}else{
		line = parseInt(line)
		total += line;
	}
});
console.log(max)
